### DISCUSSION TITLE
Default Domain Not Being Changed

### DISCUSSION THREAD

**Discussion by Brandon4466** [shlinkio/discussions#112] [Help wanted] [answered]
* Shlink Version: 3.3.2
* PHP Version: 8.1.13
* How do you serve Shlink: Self-hosted IIS
* Database engine used: SQLite 

#### Summary

When I change the DEFAULT_DOMAIN in config/params/generated_config.php it does not change the default domain, the original domain is still listed and used as the default. Both domains are listed on the list however the old one still shows as default.

#### Current behavior

Default domain not changing.

#### Expected behavior

Default domain should be updated to the one input.

#### How to reproduce

Attempt to change the domain

**Comment by acelaya** [comment, marked as answer]
If you need to change any configuration option, use `vendor/bin/shlink-installer set-option`.

See https://shlink.io/documentation/command-line-interface/installation-tool/#set-option and https://shlink.io/documentation/troubleshooting/#how-do-i-change-config-options-after-installing-shlink for more information.
